docs/drafts/developer.md

```diff
@@ -16,27 +30,7 @@ The latest version of Operator Controller can be installed with the following co
 curl -L -s https://github.com/operator-framework/operator-controller/releases/latest/download/install.sh | bash -s
 ```
 
-### Additional setup on Macintosh computers
-On Macintosh computers some additional setup is necessary to install and configure compatible tooling.
-
-#### Install Homebrew and tools
-Follow the instructions to [installing Homebrew](https://docs.brew.sh/Installation) and then execute the following to install tools:
-
-```sh
-brew install bash gnu-tar gsed
-```
-
-#### Configure your shell
-Modify your login shell's `PATH` to prefer the new tools over those in the existing environment.  This example should work either with `zsh` (in $HOME/.zshrc) or `bash` (in $HOME/.bashrc):
-
-```sh
-for bindir in `find $(brew --prefix)/opt -type d -follow -name gnubin -print`
-do
-  export PATH=$bindir:$PATH
-done
-```
-
-### Running on the cluster
+### Manual step-by-step installation
```

Comment:
Similar comment here, if we are using titlecase for all the headings, this should be "Manual Step-by-Step Installation"

Reply:
done in 7f54af6c